src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -2361,12 +2361,23 @@ public static string LastWriteTimeString(PSObject instance)
                     // non-existing targets on either Windows or Linux.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
New-Item: symlink with absolute/relative path test.Symlink with relative path to existing directory behaves like a directory
```powershell
Expected the value to have type [System.IO.DirectoryInfo] or any of its subtypes, but got C:\Users\VssAdministrator\AppData\Local\Temp\d4090637-869c-43f9-a060-9e324846e3c1\someDirLinkRelative with type [System.IO.FileInfo].
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\New-Item.Tests.ps1: line 289
289:         Get-Item someDirLinkRelative | Should -BeOfType System.IO.DirectoryInfo
```